nectar/downloaders/threaded.py

```diff
@@ -65,7 +60,7 @@ class HTTPThreadedDownloader(Downloader):
     HTTP, HTTPS and proxied download requests by the server.
     """
 
-    def __init__(self, config, event_listener=None):
+    def __init__(self, config, event_listener=None, retries=DEFAULT_RETRIES):
```

Comment:
We should document this one way or another. Perhaps just add a doc block to this function.